standard_autoencoder.py

```diff
@@ -102,18 +100,32 @@ def get_loss(self):
                         labels=X_binary),
                     axis=1),
                 axis=0)
+            # upweight binary loss by the binary loss weighting. 
+            binary_loss = self.binary_loss_weighting * binary_loss
 
         if len(self.continuous_feature_idxs) == 0:
             continuous_loss = tf.zeros(1)
         else:
-            continuous_loss = .5 * (
-                tf.reduce_mean(
-                    tf.reduce_sum(
-                        tf.square(X_continuous - Xr_continuous), 
-                        axis=1),
-                    axis=0))
+            if self.learn_continuous_variance:
```

Comment:
Curious: with a one layer network do we learn roughly the same variance as PPCA?

Reply:
in general the estimates for variance looked on par though not identical. not sure about one-layer specifically.